lib/components/term.js

```diff
@@ -3,13 +3,15 @@ import React from 'react';
 import {Terminal} from 'xterm';
 import * as fit from 'xterm/lib/addons/fit/fit';
 import * as webLinks from 'xterm/lib/addons/webLinks/webLinks';
+import * as winptyCompat from 'xterm/lib/addons/winptyCompat';
```

Comment:
Does this not need `xterm/lib/addons/winptyCompat'/winptyCompat` to link to a certain file, like with webLinks?

Reply:
😮 Good catch! Thank you!